packages/pds-ember/app/styles/pds/components/code-block/index.scss

```diff
@@ -1,5 +1,7 @@
 @use "config" as *;
 
+@use "reset";
```

Comment:
This is needed to correct the appearance of Storybook Doc block code snippets.

**BEFORE:**
![CleanShot 2020-10-15 at 18 23 44](https://user-images.githubusercontent.com/545605/96195883-9a276380-0f13-11eb-82d5-aa0cd7a38f8e.png)

**AFTER:**
![CleanShot 2020-10-15 at 18 23 52](https://user-images.githubusercontent.com/545605/96195899-a3183500-0f13-11eb-952e-b86ec80a8eee.png)